content/docs/user-guide/how-to/add-output-to-stage.md

```diff
@@ -0,0 +1,69 @@
+# Add Output to Stage
+
+There are situations where you might have forgotten to specify one of the
+<abbr>outputs</abbr> for a stage while creating it, and then later want to add
+an output to that stage.
+
+Running a stage to specify additional output is an expensive step, as it
+requires time and resources to execute the command. Follow the steps listed
+below to add an output to the stage without running it again.
+
+Let's first create a stage for the example <abbr>project</abbr>:
+
+```dvc
+$ dvc run -n prepare \
+          -d src/prepare.py \
+          -o data/train \
+          python src/prepare.py
+```
+
+You can check stage entry in `dvc.yaml`:
+
+```dvc
+$ cat dvc.yaml
+stages:
```

Comment:
Yeah in fact the dvc.yaml file is what really matter here. Using `dvc run` is not required and in fact we encourage users to make dvc.yaml files manually and only consider `run` a helper. The `run` command block could stay but as a note after the dvc.yaml block.